src/TWAlgorithms.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2017-2022 Trust Wallet.
```

Comment:
can you create an `algorithm` folder under `src`? 
- rename `SortHelpers.h` to more explicit file name without `Helper` (avoid helper / util / common as much as possible)
- No need to Add `TW` prefix for `TWAlgorithms.h` if this header is not exposed to C, also rename to `erase.h` or other explicit file name

Reply:
> can you create an `algorithm` folder under `src`?
> 
> * rename `SortHelpers.h` to more explicit file name without `Helper` (avoid helper / util / common as much as possible)
> * No need to Add `TW` prefix for `TWAlgorithms.h` if this header is not exposed to C, also rename to `erase.h` or other explicit file name

Last commits I made the following change, please let me know if it's enough:

- create algorithm folder in lowercase
- rename TWAlgorithms.h/TWAlgorithmsTests.cpp to Erase.h/EraseTests.cpp
- rename SortHelpers.h/SortHelpersTests.cpp to SortCopy.h/SortCopyTests.cpp
- keep lowercase coding style in Erase.h because it follows the standard c++ algorithm implementation.

Reply:
Yeah, lower file name and snake case work too, when c++ 20 is ready, we don't need them, correct?

Reply:
> Yeah, lower file name and snake case work too, when c++ 20 is ready, we don't need them, correct?

Yes